visualizations/frontend/filters/v2/TDatePicker.vue

```diff
@@ -89,6 +89,10 @@ export default {
       type: String,
       default: "",
     },
+    isRequired: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
I also thought that perhaps this could be derived from `default-value` being given. I presume the default-value is there as a sort of indication that the date picker needs a value. Perhaps if a default value is provided, we should not show the reset button. However, if you've already thought on that, you can resolve this comment and move forward

Reply:
Yes, I was also confused between both approaches, maybe in future we can have a default value that is clearable as well.